api/Node.json

```diff
@@ -0,0 +1,1107 @@
+{
+  "api": {
+    "Node": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node",
+        "support": {
+          "webview_android": {
+            "version_added": true,
+            "notes": "WebKit and old versions of Blink incorrectly do not make <code>Node</code> inherit from <code>EventTarget</code>."
+          },
+          "chrome": {
+            "version_added": true,
+            "notes": "WebKit and old versions of Blink incorrectly do not make <code>Node</code> inherit from <code>EventTarget</code>."
+          },
+          "chrome_android": {
+            "version_added": null
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "1"
+          },
+          "firefox_android": {
+            "version_added": "4"
+          },
+          "ie": {
+            "version_added": "9"
+          },
+          "opera": {
+            "version_added": true,
+            "notes": "WebKit and old versions of Blink incorrectly do not make <code>Node</code> inherit from <code>EventTarget</code>."
+          },
+          "opera_android": {
+            "version_added": true,
+            "notes": "WebKit and old versions of Blink incorrectly do not make <code>Node</code> inherit from <code>EventTarget</code>."
+          },
+          "safari": {
+            "version_added": true,
+            "notes": "WebKit and old versions of Blink incorrectly do not make <code>Node</code> inherit from <code>EventTarget</code>."
+          },
+          "safari_ios": {
+            "version_added": true,
+            "notes": "WebKit and old versions of Blink incorrectly do not make <code>Node</code> inherit from <code>EventTarget</code>."
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "baseURI": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node/baseURI",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
```

Comment:
All Chrome versions should be `true`. You can verify support of all of Node's members [in the IDL file](https://cs.chromium.org/chromium/src/third_party/WebKit/Source/core/dom/Node.idl?q=baseURI+file:%5Esrc/third_party/WebKit/Source/+package:%5Echromium$&dr=C&l=39).

Reply:
👍